### PR TITLE
fix/RSS-ECOMM-1_19: Create registration and login pages, add base layout

### LINKED ISSUE #67: RSS-ECOMM-2_19: Implement Direct Navigation Links to Main, Login, and Registration Pages for Unauthorized Users (15 points) 🚀
Implement direct navigation for unauthorized users to the main, login, and registration pages. This means that users should be able to access these pages directly by typing the respective URLs into the address bar of their web browser, without needing to navigate through other pages of the application first. Ensure these direct access links are working properly and take users to the correct pages. 🔗

**Acceptance Criteria**
- [x] Users can directly access the main page by typing its URL into the browser's address bar. 🏠
- [x] Users can directly access the login page by typing its URL into the browser's address bar. 🔒
- [x] Users can directly access the registration page by typing its URL into the browser's address bar. 📝
- [x] Direct access to these pages does not cause any errors or issues with the application's functionality. ✔️
- [x] Navigation using the implemented routing is smooth, without page refreshes (when applicable, e.g., in a Single Page Application). ⚙️
- [x] Navigational features of the browser (such as the back and forward buttons) work correctly after using these direct access links. 🔄
**Webinar Resources:**

- [RS School 2023Q1 Classes, SPA. Part 1.](https://www.youtube.com/watch?v=sOlzPxs_Lg4)
- [RS School 2023Q1 Classes, SPA. Part 2.](https://www.youtube.com/watch?v=njk5inZIwCc)
As Willie Nelson puts it, "On the road again, I just can't wait to get on the road again." Let's make sure our users are always ready and eager to get 'on the road' to our application's pages with perfect direct navigation! 🎵🛣️

### PR DESCRIPTION
**Pull request by Gromov-Roman**
#### 🤔 This is a ...

- [x] 🌟 New task

#### Description

- **Brief Overview:**
Implement direct navigation for unauthorized users to the main, login, and registration pages. This means that users should be able to access these pages directly by typing the respective URLs into the address bar of their web browser, without needing to navigate through other pages of the application first. Ensure these direct access links are working properly and take users to the correct pages. 🔗
- **Implementation Approach:**
- [x] Users can directly access the main page by typing its URL into the browser's address bar. 🏠
- [x] Users can directly access the login page by typing its URL into the browser's address bar. 🔒
- [x] Users can directly access the registration page by typing its URL into the browser's address bar. 📝
- [ ] Direct access to these pages does not cause any errors or issues with the application's functionality. ✔️
- [x] Navigation using the implemented routing is smooth, without page refreshes (when applicable, e.g., in a Single Page Application). ⚙️
- [ ] Navigational features of the browser (such as the back and forward buttons) work correctly after using these direct access links. 🔄

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.
- [x] 👌 Have at least 1 approve.
- [x] 🌳 Branch from `develop`/`release/...` and target to `develop`/`release/...`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,16 +24,17 @@
...
     "@commercetools/sdk-client-v2": "^2.5.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/icons-material": "^5.15.16",
     "@mui/material": "^5.15.16",
     "@mui/x-date-pickers": "^7.3.2",
+    "react": "^17.0.0 || ^18.0.0",
+    "react-dom": "^17.0.0 || ^18.0.0",
```
Comment (JsPowWow) [inline package.json]: Why this "^17.0.0 || ^18.0.0" ? Should stay as before  - as in the 27,28 lines (did You sync your branch with latest develop(?))

File: src/constants/ui.const.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const drawerWidth = 240;
+
+export default drawerWidth;
```
Comment (JsPowWow) [inline src/constants/ui.const.ts:4]: What is especial in drawerWidth that it is exported by default ?

File: package.json (hunk 1 above)
Comment (Gromov-Roman) [reply package.json]: sooooo strange

File: src/constants/ui.const.ts (hunk 2 above)
Comment (Gromov-Roman) [reply src/constants/ui.const.ts:4]: just to mute eslint (defaulr export is preferable), I'll add todo comment here

File: package.json (hunk 1 above)
Comment (Maksim99745) [reply package.json]: I had the same problem, also found it weird

**Review by Maksim99745 — APPROVED** [review]

**Pull request merged**